core/app/models/spree/item_adjustments.rb

```diff
@@ -58,9 +58,12 @@ def update
       @item.included_tax_total = tax.select(&:included?).map(&:update!).compact.sum
       @item.additional_tax_total = tax.reject(&:included?).map(&:update!).compact.sum
 
-      item_cancellation_total = adjustments.select(&:cancellation?).map(&:update!).compact.sum
+      adjustments.select(&:cancellation?).map(&:update!)
 
-      @item.adjustment_total = @item.promo_total + @item.additional_tax_total + item_cancellation_total
+      @item.adjustment_total = @item.adjustments.
+        eligible.
+        reject(&:included?).
```

Comment:
e.g. VAT in Europe is an "included" tax whereas U.S. sales tax is an "additional" tax, in Solidus terms.